Readme.md

```diff
@@ -282,36 +304,37 @@ program
 program.parse(process.argv);
 ```
 
- An `Array` is used for the value of a variadic argument.  This applies to `program.args` as well as the argument passed
- to your action as demonstrated above.
+The variadic argument is passed to the action handler as an array.
```

Comment:
Ah, rereading the diff I see the point of the `program.args`. Should add something back.